scripts/index.js

```diff
@@ -0,0 +1,117 @@
+const state = {
+    temp: 82,
+    sky: "Sunny",
+    city: "Emerald City"
+};
+
+const skys = [`☁️ ☁️ ☁️ ☀️ ☁️ ☁️`,
+    `☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️`,
+    `🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧`,
+    `🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨`
+]
+
+const landscapes = [
+    `🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂`,
+    `🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷`,
+    `🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃`,
+    `🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲`
+]
+
+const incrementTemp = () => {
+    let temp = document.getElementById("temp");
+    state.temp += 1;
+    temp.innerHTML = state.temp;
+    const landscapeIdx = getLandscapeIdx()
+
+    const landscapeImg = document.getElementById("landscapeImg");
+    landscapeImg.innerHTML = landscapes[landscapeIdx]
+}
+
+const decrementTemp = () => {
+    const temp = document.getElementById("temp");
+    state.temp -= 1;
+    temp.innerHTML = state.temp;
+
+    const landscapeIdx = getLandscapeIdx()
+    const landscapeImg = document.getElementById("landscapeImg");
+    landscapeImg.innerHTML = landscapes[landscapeIdx]
+}
+
+const getLandscapeIdx = () => {
+    if (state.temp >= 80) {
+        return 0
+    } else if (state.temp >= 70) {
+        return 1
+    } else if (state.temp >= 60) {
+        return 2
+    } else {
+        return 3
+    }
+}
+
+const getSkyIdx = () => {
```

Comment:
This has the same possible drawbacks as the ground helper. Consider storing enough information with the sky strings to allow for a more data-driven approach where both the options in the select and their resultant use to lookup the sky strings are intrinsically related.